pom.xml

```diff
@@ -193,6 +193,17 @@
                 <artifactId>websocket-jetty-server</artifactId>
                 <version>${jetty.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-unixsocket</artifactId>
+                <version>${jetty.version}</version>
+                <type>pom</type>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-unixsocket-server</artifactId>
+                <version>${jetty.version}</version>
+            </dependency>
```

Comment:
It's probably better to keep those strange jetty & websocket related libraries as there's already a section for them

Reply:
Sure.